scopes/workspace/workspace/workspace.ts

```diff
@@ -476,10 +477,10 @@ export class Workspace implements ComponentFactory {
     const results: Array<{ extensionId: string; results: SerializableResults }> = [];
     await mapSeries(onAddEntries, async ([extension, onAddFunc]) => {
       const onAddResult = await onAddFunc(component);
-      await this.graphql.pubsub.publish(ComponentAdded, { componentAdded: { component } });
       results.push({ extensionId: extension, results: onAddResult });
     });
 
+    await this.graphql.pubsub.publish(ComponentAdded, { componentAdded: { component } });
```

Comment:
this published the `ComponentAdded` event many times in the loop, moved it outside to publish only once.
@guysaar223  - what do you think?